PotionCraft_SortBookmark/SortBookmark.cs

```diff
@@ -141,26 +148,32 @@ public void SortAllBookmark()
                 Debug.Log($"将{emptyPbs.Count}个空书签堆放在了最后一个轨道");
             }
 
+            // Trigger the bookmarks rearranged method to reorganize the savedRecipes list to match the new bookmark order
+            markCtl.CallOnBookmarksRearrangeIfNecessary(preSortBookmarkList);
```

Comment:
Called the same method used when normally moving bookmarks which automatically rearranges recipe pages to match bookmark order and will allow my mods to rearrange any stored indexes as well.